orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -291,7 +291,7 @@ def _extract_schema(data):
     elif isinstance(data, torch.Tensor):
         return _TensorStub(dtype=str(data.dtype), shape_dims=len(data.size()))
 
-    if isinstance(data, abc.Sequence):
+    if isinstance(data, abc.Sequence) and not isinstance(data, str):
```

Comment:
Shall we also handle byte strings here?

Reply:
do you know whether onnx support them? we could do on a separate PR to unblock 1.8.1

Reply:
All pipelines are green